lib/docassemble/docassemble_api_interface.js

```diff
@@ -204,16 +204,17 @@ da_i.loop_to_create_project = async function( creation_options ) {
   let wait_seconds = creation_options.wait_seconds || 5;
 
   let project_name = null;
-  let name_incrementor = 0;
+  // Unlikely, but 2 devs could run locally and create the same Project name
```

Comment:
`Date.now` goes to milisecond precision, and IMO it's almost impossible to get the same milisecond. But since the code already handles it, it's fine to keep it there to simplfy the PR